hols/todo/infra/multi-cluster/scripts/set_alias.template.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/bash
+# Copyright (c) 2025 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kubectx ${cluster}=context-${cluster_id_11}
```

Comment:
A script with one command ?!

Reply:
Terraform iterates through it. I'm open for a better alternative

Reply:
I'm not a terraform expert, I'll have to trust you on that one.